scripts/db_migrator.py

```diff
@@ -199,17 +210,16 @@ def version_1_0_3(self):
         if self.asic_type == "mellanox":
             if self.mellanox_buffer_migrator.mlnx_migrate_buffer_pool_size('version_1_0_3', 'version_1_0_4') and self.mellanox_buffer_migrator.mlnx_migrate_buffer_profile('version_1_0_3', 'version_1_0_4'):
                 self.set_version('version_1_0_4')
-        else:
-            self.set_version('version_1_0_4')
 
+        self.set_version('version_1_0_4')
         return 'version_1_0_4'
 
     def version_1_0_4(self):
         """
-        Current latest version. Nothing to do here.
+        Current latest version. Always perform copp table migration at current latest version
         """
         log.log_info('Handling version_1_0_4')
-
+        self.migrate_copp_table()
         return None
```

Comment:
Please upgrade to version_1_0_5 and create function version_1_0_5 with previous code in version_1_0_4. You need to mark that upgrade is done.

Reply:
Had a discussion with @yxieca . @dgsudharsan, can we do it as part of an outer function so that this is called always instead of copying to each later versions?

Reply:
sure. after offline discussion. I think we need to put 'always apply at latest version' operations after the main upgrade loop.